src/common/services/translator.ts

```diff
@@ -132,6 +132,10 @@ export class Translator {
     public translate: I18nTyped = this._translate;
     private locale: string = "en";
 
+    constructor(defaultLocale = "en") {
```

Comment:
Is this change necessary? The Dependency Injection Container will not automatically create a `Translator` instance with a constructor parameter, so this would need to be an explicit `new Translator("fr")` (for example) from `di.ts` ... but this is currently not the case:

https://github.com/readium/readium-desktop/blob/aacfe6bbd33db5623a01cfb2939713b6015c8790/src/main/di.ts#L167

Reply:
you're right c2ca422276e16d13109655e3cec416cacbad3969